index.html

```diff
@@ -1615,6 +1615,36 @@ <h2>A project for first-time contributions</h2>
 
         </div>
         <!-- ________ Contributor's card END ________  -->
+		
+		
+		<!-- ________ YekKhai' card START ________  -->
+        <div class="card">
+
+            <p class="name">Ch'ng Yek Khai</p>
+            <p class="contact">
+                <i class="fa fa-twitter"></i>
+                <a href="https://twitter.com/ykchng97" target="_blank">@ykchng</a>
+            </p>
+            <p class="about
```

Comment:
You are missing the closing quotation mark for the class and the `>` for the `p` opening tag. This means the text that follows is not in the right place. To fix this you simply need to follow the suggestion below. Then try it out to make sure your card shows correctly
```suggestion
            <p class="about">
```